### PR TITLE
Lisbeth Orders

### PR DESCRIPTION
**Pull request by cheesegoldfish**

### CODE REVIEW

File: LeveGenerator.cs [hunk 1]
```diff
@@ -38,6 +43,37 @@ public static void Generate(LeveDatabase db, ObservableCollection<Leve> currentO
             }
         }
 
+        private static string WriteLisbeth(LeveDatabase db, ObservableCollection<Leve> currentOrder) {
+            var lisbethOrder = $@"
+        <Lisbeth Json=""[";
+
+            foreach (var leve in currentOrder.OrderBy(i => i.Level))
+            {
+                var amount = (leve.Repeats > 0) ? leve.NumItems * (leve.Repeats + 1) : leve.NumItems;
```
Comment (zzi-zzi-zzi) [inline LeveGenerator.cs:52]: so it looks like we are doing all of the crafting at the start, what if someone is using this to level? 

Is it better to craft, then turn in, then craft etc ?

File: generator/generator.txt [hunk 2]
```diff
@@ -122,6 +117,8 @@ namespace SaintCoinach.Cmd.Commands {
             public string Name { get; set; }
             public string Classes { get; set; }
             public int Level { get; set; }
+            public int ExpReward { get; set; }
```
Comment (zzi-zzi-zzi) [inline generator/generator.txt:120]: These don't seem to be assigned here. can you include assigning values here or did you assign them manually?

Comment (cheesegoldfish) [reply generator/generator.txt:120]: They should be assigned on lines 63 and 64?

**Comment by cheesegoldfish** [comment]
realized the database.json is fubar, don't use.

**Pull request closed without merging**